docs/demo.rst

```diff
@@ -0,0 +1,22 @@
+A Simple Demo for GluonNLP
+==========================
+
+Introduction
+------------
+
+This demo is using MMS Server
+
+Demo
+----
+
+Please input the following into the text box:
+
+.. raw:: html
+
+   ["Positive sentiment", "Negative sentiment"]
```

Comment:
We shall have a drop down list of a few pre-defined example inputs, and another text box to support custom user input.